logic/labrynna.go

```diff
@@ -158,13 +163,16 @@ var labrynnaNodes = map[string]*Node{
 	"ridge base past east": Or("target carts",
 		And("lynna city", Or("feather", "ages"), "mermaid suit"),
 		And("ridge mid past", "feather", "brother emblem"),
-		And("ridge mid present", "ages"),
+		"rolling ridge east tree",
+		And("ridge base present", "ages"),
 		And("ridge base past west", Or("flippers", Hard("jump 3")))),
 	"ridge base past west": Or(
 		And("ridge base present", "echoes"),
-		And("ridge base past east", Or("flippers", Hard("jump 3")))),
+		And("ridge base past east", Or("flippers", "bomb jump 2"))),
```

Comment:
I didn't realize that you could bomb jump into D6 past. IMO anything requiring jumping into D6 past should be hard logic, though.